src/runtimes/node/dynamic_imports/parser.js

```diff
@@ -26,21 +26,50 @@ const getAbsoluteGlob = ({ basePath, globNodes, resolveDir }) => {
   return absoluteGlob
 }
 
+// Returns GLOB_WILDCARD for AST nodes that are accepted as part of a dynamic
+// expression and convertable to a wildcard character. This determines whether
+// we convert an expression or leave it alone. For example:
+//
+// - `./files/${someVariable}`: Convert `someVariable` to `*`
+// - `./files/${[some, array]}`: Don't convert expression
+//
+// The following AST nodes are converted to a wildcard:
+//
+// - CallExpression: `someFunction()`
+// - ConditionalExpression: `someCond ? someValue : otherValue`
+// - Identifier: `someVariable`
+// - MemberExpression: `someArray[index]` or `someObject.property`
+const getWildcardFromASTNode = (node) => {
+  switch (node.type) {
+    case 'CallExpression':
+    case 'ConditionalExpression':
+    case 'Identifier':
+    case 'MemberExpression':
+      return GLOB_WILDCARD
+
+    default:
+      throw new Error('Expression member not supported')
```

Comment:
This is handled in the `parseExpression` `try/catch` correct?

Reply:
Yes, exactly!